apps/teams-test-app/src/components/privateApis/CopilotAPIs.tsx

```diff
@@ -23,11 +23,34 @@ const CopilotAPIs = (): ReactElement => {
       },
     });
 
+  const SendCustomTelemetryData = (): ReactElement =>
+    ApiWithTextInput<{
+      name: copilot.Stage;
+      timestamp: number;
+    }>({
+      name: 'sendCustomTelemetryData',
+      title: 'sendCustomTelemetryData',
+      onClick: {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        validateInput: (_input) => {},
+        submit: async (input) => {
+          const result = await copilot.customTelemetry.sendCustomTelemetryData(input.name, input.timestamp);
+          return JSON.stringify(result);
+        },
+      },
+      defaultInput: JSON.stringify({ name: copilot.Stage.STAGE_E, timestamp: Date.now() }),
+    });
+
   return (
-    <ModuleWrapper title="Copilot.Eligibility">
-      <CheckCopilotEligibilityCapability />
-      <GetEligibilityInfo />
-    </ModuleWrapper>
+    <>
+      <ModuleWrapper title="Copilot.Eligibility">
+        <CheckCopilotEligibilityCapability />
+        <GetEligibilityInfo />
+      </ModuleWrapper>
+      <ModuleWrapper title="Copilot.CustomTelemetry">
```

Comment:
nit: for some reason, I am not a fan of the term `Custom` to be honest. There are too many terms contains "Custom" in hubsdk...but feel free to use as this if it is proper.

Reply:
They are custom telemetry data points, as they don't follow the existing apis.